addons/account/models/account_move.py

```diff
@@ -510,6 +510,14 @@ def _sequence_fixed_regex(self):
         compute='_compute_is_being_sent'
     )
 
+    move_sent_values = fields.Selection(
+        selection=[
+            ('sent', 'Sent'),
+            ('not_sent', 'Not Sent'),
+        ],
+        string='Sent',
+        compute='compute_move_sent_values'
```

Comment:
Add the coma at the end of the compute, so that if someone need to add something later he don't break the commit history :smile: And the parenthesis should be one tab left :eyes: